deploy-retool

```diff
@@ -242,13 +246,25 @@ fi
 # Due to a race condition with migration runner, check if api container stays up for 20 seconds.
 WAITED=0
 while [ $WAITED -lt 20 ]; do
-    if ! api_running; then
+    if ! api_container_up; then
         $MAYBE_SUDO docker-compose up -d > /dev/null 2>&1
     fi
     WAITED=$((WAITED + 1))
     sleep 1
 done
 
+WAITED=0
+# one request every three seconds. It takes about 40 seconds for the api to get ready so 1 minute should be enough margin
+while [ $WAITED -lt 20 ] && ! api_running; do
```

Comment:
Might need to wait longer than 60s since we don't know how fast our customer's computers are.